annotations/component-annotations/src/it/issue-275/src/main/java/io/dekorate/issue275/DemoApplication.java

```diff
@@ -6,7 +6,7 @@
 import io.dekorate.component.annotation.ComponentApplication;
 
 @SpringBootApplication
-@ComponentApplication(exposeService=true)
+@ComponentApplication(exposeService=true, name="customName")
```

Comment:
I am a little bit skeptic about checking for issue #281 inside the #275 integration test.
But definitely not a blocker, more like something to note for future discussions.

Reply:
What I have done is update 2 tests - one where we use the custom name and one where the default  (`artifactId` is used)

Reply:
Ok, That's what I had done but in different tests so let's take yours ;-)

Reply:
I reverted this - there is now a separate test for #276